plugins/plugin-typescript/plugin.js

```diff
@@ -0,0 +1,32 @@
+const execa = require('execa');
+const npmRunPath = require('npm-run-path');
+const cwd = process.cwd();
+
+function typescriptPlugin() {
+  return {
+    name: '@snowpack/plugin-typescript',
+    async run({isDev, log}) {
```

Comment:
I’m really liking the evolution of this humble `run()` method. On the one hand, it’s just a wrapper around a CLI command. But on the other, it does provide a pretty clean experience to leverage libraries without Snowpack having to do to much. I’m into it!